charts/kubernetes-zfs-provisioner/templates/storageclass.yaml

```
@@ -7,6 +7,10 @@ metadata:
   name: {{ .name }}
   labels:
     {{- include "kubernetes-zfs-provisioner.labels" $ | nindent 4 }}
+{{- if .defaultClass }}
+  annotations:
+    storageclass.kubernetes.io/is-default-class: "true"
+{{- end }}
```

I think the more flexible solution is to generally allow any kind of annotation.
(I have not tested whether the suggestion below works, please double-check)
```suggestion
{{- with .annotations }}
  annotations:
    {{ toYaml . | nindent 4 }}
{{- end }}
```